responsive-web-design/tribute-page/tribute.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<title>My Tribute Page</title>
+	<link rel="stylesheet" href="styles.css">
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+
+<body>
+	<main id="main">
+		<div class='header'>
```

Comment:
```suggestion
		<div class="header">
```

try to have a convention for single/double quotes. I prefer double